pyproject.toml

```
@@ -38,6 +38,8 @@ dependencies = [
     "pydantic",
     "shellingham",
     "python-dotenv",
+    "fsspec",
+    "python-libarchive-c"
```

We were just testing this and on pypi the package is called:

```suggestion
    "libarchive-c"
```